src/hardware/motor.h

```diff
@@ -147,7 +153,7 @@ class StepperMotor {
         void setCoilB(COIL_STATE desiredState, uint16_t current = 0);
 
         // Calculates the correct PWM setting based on an input current
-        uint32_t currentToPWM(uint16_t current) const;
+        uint16_t currentToPWM(uint16_t current) const;
 
         // Sets the speed of the motor (angular speed is in deg/s)
         float speedToHz(float angularSpeed) const;
```

Comment:
This should stay as a uint32_t to prevent unneeded conversions (analogSet needs a uint32_t)